ephemeral values updates (#35906)
hashicorp/terraform: website/docs/language/values/locals.mdx

}
```

## Ephemeral values

> 

Reviewer: Can you add a beta warning here too?

> Local values will implicitly become ephemeral if an ephemeral value is

Reviewer: ```suggestion
Local values implicitly become ephemeral if you reference an ephemeral value when
```

referenced in their assignment. For example:

```hcl
variable "service_name" {

> 	type    = string

Reviewer: formatting on this one
